### ISSUE TITLE
Question on Training Steps and Cross entropy Loss for Stage 2

### ISSUE THREAD

**Issue by lxxie298** [sczhou/CodeFormer#373]
Your work is truly impressive! The clarity and depth you bring to the subject are remarkable！
Could you please let me know how many **training steps** are required in stage 2 and how small the **Cross Entropy Loss** should be to achieve normal images? 

I use the pretrained VQGAN which provided by authors to start the training of stage-2, but it seem that it could generate a normal image. I have trained 70k step and batch size is 8, learning rate is 4e-5. following is my loss figure and results:
![image](https://github.com/sczhou/CodeFormer/assets/37968886/524afe6f-cebf-47c7-9fe4-1840cb38dac3)
![image](https://github.com/sczhou/CodeFormer/assets/37968886/c2588f07-a1b9-4b26-92a8-77290cd25c23)

Please, kind-hearted person, help me. I don't know what the problem is.

**Comment by qinghaoya** [comment]
Have you found the reason? The same problem arises

**Comment by lxxie298** [comment]
> Have you found the reason? The same problem arises

No, I just run another experiments. If I have time later, I will look for the reason

**Comment by Niax23** [comment]
same question, I used the pretrained stage 1 vqgan to train the stage2 from scratch but it cannot work.

**Comment by leochliu107** [comment]
same issue, could anyone helps?
